src/exceptions/api.error.js

```diff
@@ -0,0 +1,33 @@
+export class ApiError extends Error {
+  constructor({ message, status, errors }) {
+    super(message);
+
+    this.status = status;
+    this.errors = errors;
+  }
+
+  static badRequest(message, errors) {
+    return new ApiError({
+      message,
+      errors,
+      status: 400,
+    })
```

Comment:
There is a missing semicolon at the end of this line. It's a good practice to include semicolons to avoid potential issues.